### DISCUSSION TITLE
more values

### DISCUSSION THREAD

**Discussion by alexmyczko** [alexmyczko/ruptime#18] [Ideas]
power reading
`ipmitool dcmi power reading|grep -v ^$ | head -1 |awk '{print $(NF-1)}'`

temperatures?

model of computer/disks/nic/gfx

serials

**Comment by treellama** [comment]
I'd love to use this tool to monitor the temperature of my various raspberry pis. A couple ways to do this would be to add the CPU temperature to `rload`; or to add a new `rsensors` with the CPU temperature and maybe some day hddtemp and other readings. Is either of these approaches something you would be interested in a pull request for?

**Reply by alexmyczko** [reply]
I think not so much, you can of course do that for your site, but I even consider to drop rboot atm...
